service/internal/auth/config.go

```diff
@@ -4,12 +4,13 @@ import "fmt"
 
 // AuthConfig pulls AuthN and AuthZ together
 type Config struct {
-	Enabled     bool `yaml:"enabled" default:"true" `
-	AuthNConfig `mapstructure:",squash"`
+	DeprecatedDisabled bool `yaml:"deprecatedDisabled" default:"false"`
```

Comment:
@mkleene 
We can't rename the config value. This would become a breaking change. We should leave the option as is. Add comments and docs  that this is deprecated. Then remove it in the future.

Reply:
removed it